### PR TITLE
Launch homepage

### PR DESCRIPTION
**Pull request by mattandrews**
Removes the A/B test functionality for the homepage routes, removes the `/legacy` testing path, and redirects `/home` to `/`.

### CODE REVIEW

File: config/cloudfront/live.json [hunk 1]
```diff
@@ -5651,10 +5558,68 @@
...
+                }
+            },
+            "QueryStringCacheKeys": {
+                "Items": [],
+                "Quantity": 0
+            },
+            "QueryString": false
+        },
+        "MaxTTL": 31536000,
+        "PathPattern": "/welsh/home",
```
Comment (davidrapson) [inline config/cloudfront/live.json]: How come this is getting added?

File: controllers/routes.js [hunk 2]
```diff
@@ -42,7 +42,8 @@ const routes = {
                     code: 0,
                     static: false,
                     live: true,
-                    isPostable: true
+                    isPostable: true,
+                    aliases: ['/home']
```
Comment (davidrapson) [inline controllers/routes.js]: Is this to mop up any traffic that might have linked to this?

File: config/cloudfront/live.json (hunk 1 above)
Comment (mattandrews) [reply config/cloudfront/live.json]: Any alias listed for a route automatically gets a Welsh version added too. Kind of redundant in this case but causes no harm really.

Comment (davidrapson) [reply config/cloudfront/live.json]: 👍

File: controllers/routes.js (hunk 2 above)
Comment (mattandrews) [reply controllers/routes.js]: Yeah, you see this path in analytics (so presumably in google etc) so making it properly redirect would be nice.

**Comment by davidrapson** [comment]
🚀

**Pull request merged**